internal/service/timestreaminfluxdb/generate.go

```diff
@@ -2,7 +2,6 @@
 // SPDX-License-Identifier: MPL-2.0
 
 //go:generate go run ../../generate/servicepackage/main.go
-//go:generate go run ../../generate/tags/main.go -AWSSDKVersion=2 -KVTValues -SkipTypesImp -ListTags -ServiceTagsMap -UpdateTags
```

Comment:
A contributor added this line, which generates tags methods. I have removed it because I have created my own `tags.go` file and marked it as to not be affected by file generation.